script/lib/check-node-version

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+
+APP_FOLDER="apps"
+ROOT_DIR="$APP_FOLDER/../"
+
+setupNodeEnv() {
+  echo "Attempting to switch node versions for you..."
+
+  if command -v nvm &> /dev/null
+  then
+    nvm use $1
+  else
+    echo "Please install nvm: 'brew install nvm'"
```

Comment:
Recommend we don't prescribe a version manager here. I use `fnm`, this'll make me sad – others may use other version managers. Generally, I'm not sure we should feel the need to keep this script up to date with whatever version manager is du jour.

Reply:
haha snap @rhystmills 😀

Reply:
In flexible-content we do this: https://github.com/guardian/flexible-content/blob/main/scripts/check-node-version.sh -as Jon said, not that elegant but more congenial to fnm or asdf users.